src/Providers/LighthouseGraphQLPassportServiceProvider.php

```diff
@@ -23,6 +23,16 @@ class LighthouseGraphQLPassportServiceProvider extends ServiceProvider
      */
     public function boot()
     {
+        [$publicKey, $privateKey] = [
+            Passport::keyPath('oauth-public.key'),
+            Passport::keyPath('oauth-private.key'),
+        ];
+
+        /** Don't try to boot when you can't load the passport keys. */
+        if (!file_exists($publicKey) || !file_exists($privateKey)) {
```

Comment:
What if the user has the Public and private keys as Strings in the env vars? https://laravel.com/docs/6.x/passport#deploying-passport I think using file_exists may be an issue if we are using that other method.

Reply:
I have changed it so it is not needed anymore.